react-components/src/architecture/concrete/pointsOfInterest/PointsOfInterestDomainObject.ts

```diff
@@ -153,6 +157,12 @@ export class PointsOfInterestDomainObject<PoiIdType> extends VisualDomainObject
     this.notify(Changes.selected);
   }
 
+  public async updatePointOfInterest(
+    poi: PointOfInterest<PoiIdType>
+  ): Promise<PointsOfInterestInstance<PoiIdType>> {
+    return (await this._poisCache.savePointsOfInterest([poi]))[0];
+  }
+
```

Comment:
As the method within the poisCache is "save", then it is not better to use the same naming?
Or both "update" or both "save...." ?